extensions/usage-metrics-dashboard/app.R

```diff
@@ -274,25 +275,82 @@ ui <- function(request) {
 server <- function(input, output, session) {
   # Set up Connect client; handle error if Visitor API Key integration isn't
   # present.
+
+  publisher_client <- connect()
+
+  selected_integration_guid <- reactiveVal(NULL)
+  observeEvent(input$auto_add_integration, {
+    print("Clicked auto add integration button")
```

Comment:
Should we clear out prints?

Reply:
Looks like there are a few more in here too.

Reply:
Yes, clearing all but a few out.